editor/editor_themes.cpp

```diff
@@ -677,7 +677,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_constant("item_margin", "Tree", 3 * default_margin_size * EDSCALE);
 	theme->set_constant("button_margin", "Tree", default_margin_size * EDSCALE);
 	theme->set_constant("draw_relationship_lines", "Tree", 0);
-	theme->set_constant("scroll_border", "Tree", default_margin_size * EDSCALE);
+	theme->set_constant("scroll_border", "Tree", 40);
```

Comment:
Is the removal of `EDSCALE` intentional here? The dragging margin should be independent of the editor scale.

Reply:
Lol, I don't remember. I guess so but I might have made a mistake I am not sure.
I'll check when rebasing.

Reply:
Rebase still pending :)

Reply:
Done